src/components/SettlementButton.js

```diff
@@ -1,21 +1,29 @@
+import React, {useEffect, useMemo} from 'react';
 import _ from 'underscore';
-import React from 'react';
 import PropTypes from 'prop-types';
 import {withOnyx} from 'react-native-onyx';
-import ButtonWithDropdownMenu from './ButtonWithDropdownMenu';
-import * as Expensicons from './Icon/Expensicons';
-import Permissions from '../libs/Permissions';
-import ONYXKEYS from '../ONYXKEYS';
+
```

Comment:
I also like this way of organizing imports, but it seems like the standard here is to not have empty lines between imports